consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/headervalidationrules/SignerRateLimitValidationRuleTest.java

```diff
@@ -1,18 +0,0 @@
-/*
- * Copyright 2018 ConsenSys AG.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
- * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations under the License.
- */
-package tech.pegasys.pantheon.consensus.clique.headervalidationrules;
-
-public class SignerRateLimitValidationRuleTest {
-
-  // Implicitly conducted by NodeCanProduceNextBlockTest.
```

Comment:
Instead of deleting the test can we add a small test for this validation

Reply:
No.

Reply:
...but if you have a test in mind, then don't let me stop you ;)